grow_cluster_test.py

```diff
@@ -107,9 +107,10 @@ def get_stress_cmd(self):
                 (duration, threads, population_size, ip))
 
     def grow_cluster(self, cluster_target_size, stress_cmd):
-        # 60 minutes should be long enough for adding each node
+        # 24 * 60 minutes should be long enough for adding each node
+        # we may deal with large database for the benchmarks
         nodes_to_add = cluster_target_size - self._cluster_starting_size
-        duration = 60 * nodes_to_add
+        duration = 24 * 60 * nodes_to_add
```

Comment:
It's not good to change the timeout for all tests, how about add a new parameter in configure file, the default timeout is still 60 mins, and only assign the timeout to 24 hour in data_dir/benchmark_patch.diff ?